docs/docs/administer/logs.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Logs
+sidebar_position: 8
+description: Firezone's WireGuard®-based remote access platform stores debug logs and configuration, and audit logs.
+---
+
+:::note
+This guide is written for Docker based deployments of Firezone
+:::
+
+## What Firezone Logs
+
+Docker deployments of Firezone consist of 3 running containers:
+
+| Container         | Function      | Example logs                                  |
+|-------------------|---------------|-----------------------------------------------|
+| ubuntu-firezone-1 | Web portal    | HTTP requests received and responses provided |
+| ubuntu-postgres-1 | Database      |                                               |
+| ubuntu-caddy-1    | Reverse proxy |                                               |
+
+Each container stores logs as a JSON file on the host machine. These files are found at
+`var/lib/docker/containers/{CONTAINER_ID}/{CONTAINER_ID}-json.log` and can be shown through the
+`docker logs {CONTAINER}` command.
```

Comment:
This section is applicable only to Linux (and looks to be specific to Ubuntu in particular).

It's probably worth rewriting it using `docker compose logs` instead of `docker logs`. Compose abstracts most of the system-specific parts.

Users need to be aware that `docker compose` commands should either be run by within the Firezone root directory at `$HOME/.firezone` or be run with the `-f $HOME/.firezone/docker-compose.yml` flag so that Docker Compose knows where the compose file is.

I'd probably omit the Docker-specific logging details (directory where stored, etc) and just point them to the relevant Docker docs.